quick_start_new.md

```diff
@@ -0,0 +1,34 @@
+## Setup
+Compatible with Python versions 3.8.10 and higher
+### Install dependencies
+```bash
+# Create an virtual environment 
+# (optional but recommended)
+$ python -m venv venv 
+$ source venv/bin/activate
+# install dependencies
+(venv) $ pip install -r requirements_quick_start.txt
+```
+
+## Create config file
+```bash
+python ver_cli.py create_sources_file quickstart
+python ver_cli.py add_csv quickstart quickstart <DIR>
+```
+`<DIR>` is an absolute path to your dataset.
+
+## Build Data Profiles
+```bash
+python ver_cli.py profile quickstart output_profiles_json --store_type 3
```

Comment:
Have you tried this line? In a quick glance, I don't think it will run (correctly).

Reply:
It's not the profile command fault. There is a little error in the yml writer.